docs/user-guides/observability-ui-plugins.md

```
@@ -267,18 +251,17 @@ spec:
 
 #### Feature List
 
-| __Feature__         | __Description__                                                                                                          | __Support Level__    |
-| ------------------- | ------------------------------------------------------------------------------------------------------------------------ | -----------------    |
-| `acm-alerting`      | Adds alerting UI to multi-cluster view. Configures proxies to connect with any Alertmanager and Thanos Query API.        | Dev Preview          |
-| `incidents`         | Adds incidents UI to `Observe` section of OpenShift Console Platform. Deploys the [Cluster Health Analyzer](https://github.com/openshift/cluster-health-analyzer) and configures proxies in the plugin to connect with it. | General Availability |
+| **Feature**         | **Description**                                                                                                                                                                                                                              | **Support Level**    |
+| ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------- |
+| `acm-alerting`      | Adds alerting UI to multi-cluster view. Configures proxies to connect with any Alertmanager and Thanos Query API.                                                                                                                            | Dev Preview          |
+| `incidents`         | Adds incidents UI to `Observe` section of OpenShift Console Platform. Deploys the [Cluster Health Analyzer](https://github.com/openshift/cluster-health-analyzer) and configures proxies in the plugin to connect with it.                   | General Availability |
 | `perses-dashboards` | Adds perses UI to `Observe` section of OpenShift Console Platform. Configures proxies to connect with a Perses instance. Installs Accelerator Perses Dashboard and Accelerator Perses Datasource. See details [here](./perses-dashboards.md) | Dev Preview          |
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)